### PR TITLE
Index auto

### PR DESCRIPTION
**Pull request by Gliu1211**
review please

### CODE REVIEW

File: src/main/java/frc/robot/commands/OrganizeIndexCMD.java [hunk 1]
```diff
@@ -27,15 +28,13 @@ public void initialize() {
...
   @Override
   public void execute() {
-    if ( !(index.getBallPosition(1)) && index.getBallPosition(2) ){
-      isDone = false;
-      index.setSpeedTower(0.75);
-    } else {
-      isDone = true;
-    }
+
+
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/commands/OrganizeIndexCMD.java]: If `execute()` is empty, then why does this command (still) exist?

File: src/main/java/frc/robot/commands/indexRun.java [hunk 2]
```diff
@@ -9,6 +9,7 @@
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import frc.robot.Constants;
+import frc.robot.STATES;
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/commands/indexRun.java]: Remove unused import.

File: src/main/java/frc/robot/commands/indexRun.java [hunk 3]
```diff
@@ -32,7 +33,8 @@ public void initialize() {
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    
+
+
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/commands/indexRun.java]: Remove unnecessary whitespace.

File: src/main/java/frc/robot/RobotContainer.java [hunk 4]
```diff
@@ -30,7 +30,7 @@
 import frc.robot.commands.zeroHeading;
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/RobotContainer.java]: Remove unused import.

File: src/main/java/frc/robot/STATES.java [hunk 5]
```diff
@@ -0,0 +1,9 @@
+package frc.robot;
+
+public enum STATES{
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/STATES.java]: `STATES` -> `States`

File: src/main/java/frc/robot/subsystems/IndexSubsystem.java [hunk 6]
```diff
@@ -5,11 +5,11 @@
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.robot.Constants;
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/subsystems/IndexSubsystem.java:9]: Remove unused import.

File: src/main/java/frc/robot/subsystems/IndexSubsystem.java [hunk 7]
```diff
@@ -24,9 +24,18 @@
   // Motors
   
   private TalonSRX tower = new TalonSRX(Constants.TOWER_INDEX_ID);
+  private DigitalInput bannerSensor1 = new DigitalInput(Constants.DIO_PORT_0);
+  private DigitalInput bannerSensor2 = new DigitalInput(Constants.DIO_PORT_1);
 
-  private BannerSensorSubsystem bannerSensorControl= new BannerSensorSubsystem();
-  
+
+  public enum STATES {
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/subsystems/IndexSubsystem.java:31]: Why are there two `STATES` `enum`s in the repository? There should only be one.

Also, `STATES` should be `States`.

**Review by liam-a-wms — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/OrganizeIndexCMD.java (hunk 1 above)
Comment (Gliu1211) [reply src/main/java/frc/robot/commands/OrganizeIndexCMD.java]: Deleted execute

File: src/main/java/frc/robot/subsystems/IndexSubsystem.java (hunk 7 above)
Comment (liam-a-wms) [reply src/main/java/frc/robot/subsystems/IndexSubsystem.java:31]: Is `STATES` even being used anywhere to begin with?

File: src/main/java/frc/robot/commands/OrganizeIndexCMD.java [hunk 8]
```diff
@@ -25,14 +25,7 @@ public void initialize() {
 
   }
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/commands/OrganizeIndexCMD.java:39]: If there's no `execute()` and this command can't do anything, why is it still here?

**Review by liam-a-wms — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/indexRun.java (hunk 2 above)
Comment (KevinH45) [inline src/main/java/frc/robot/commands/indexRun.java]: Unused + STATES does not exist anymore. This code won't work.

**Review by KevinH45 — CHANGES_REQUESTED** [review]

**Review by liam-a-wms — APPROVED** [review]

**Review by KevinH45 — APPROVED** [review]
